.github/workflows/udeps.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       bundler: ${{ steps.filter.outputs.bundler }}
       cli: ${{ steps.filter.outputs.cli }}
     steps:
-      - uses: actions/checkout@v2
```

Comment:
We're still using v2 in several places though

Reply:
Do you want me to apply the migration to `v3` to all the workflows in the same PR? I found it convinient to do it in separate PRs.